test/regress/gold/cover11.txt

```diff
@@ -0,0 +1,7512 @@
+** Note: excluding toggle: 'WORK.COVER11.ARR_B(1500)' bins: BIN_0_TO_1, BIN_1_TO_0
```

Comment:
This file is too big (and also not used?)

Reply:
OK, I removed it and check only that the item is/is not part of the HTML report by grep.